app/assets/stylesheets/products.scss

```diff
@@ -0,0 +1,167 @@
+// Place all the styles related to the products controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.slider {
+  height: 540px !important;
+}
+.slides {
+  height: 500px !important;
+  min-height: 500px !important;
+  // width: 500px !important;
+  li, li.active {
+    z-index: 0 !important;
+  }
+}
+
+.waves-effect {
+  z-index: 0 !important;
+}
+
+.product-description-body {
+  margin-top: 3em;
+}
+.product-view-image {
+  margin: 0 auto;
+}
+.product-name {
+  font-size: +1;
+}
+
+.price {
+  color: #2e7d32;
+  font-size: 1.2em;
+  width: 8em;
+  .old-price{
+    color: #730E15;
+    text-decoration: line-through;
+    font-size: smaller;
+    font-weight: bold;
+  }
+} 
+
+.rating-counts {
+  width: 100%;
+  margin: 10px 0 30px;
+  float: left;
+}
+
+.counts {
+  margin: 1.1em 0 0 1em;
+  font-size: 1em;
+  /*font-family: "Arial Black";*/
+  color: #000;
+}
+
+.rating {
+  margin: 0;
+  unicode-bidi: bidi-override;
+  direction: rtl;
+  max-width: 6.5em;
+  font-size: 2em;
+}
+
+.rating > span {
+  display: inline-block;
+  position: relative;
+  //width: 1.1em;
+}
+
+.rating > span:hover:before,
+.rating > span:hover ~ span:before {
+  content: "\2605";
+  position: absolute;
+}
+///////End of rating styling///////////////////
+
+.buy-now {
+  margin-top: 3rem;
+}
+.my-icons {
+  color: $my-grey;
+}
+
+.wish-list-link {
+  color: $my-grey;
+  &:hover {
+    color: $my-green;
+  }
+  .wishlist-icon {
+  font-size: 1.8em !important;
+    padding: .2em 0;
+    vertical-align: middle;
+  }
+}
+
+
+td:has(select) {
+  background-color: red;
+}
+
+.shares {
+  margin-top: 3em;
+}
+
+.fa-facebook:hover {
+  color: #3b5998;
+}
+
+.fa-twitter:hover {
+  color: #4099FF;
+}
+
+.fa-instagram:hover {
+  color: #3f729b;
+}
+
+.fa-pinterest-p:hover {
+  color: #c92228;
+}
+//
+//.card-reveal {
+//  opacity: 0.9;
+//}
+//
+//.price-rating {
+//  width: 100%;
+//  margin: 0 0 30px;
+//  float: left;
+//}
+#description-tabs {
+  margin-bottom: 2em;
+  .tabs {
+    overflow-x: hidden !important;
+    li {
+      // flex-grow: 0;
+      // webkit-flex-grow: 0;
+      border-bottom: 3px solid #bababa;
+      a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 4